CONTRIBUTING.md

```diff
@@ -37,381 +48,295 @@ to sign the CLA. Therefore,
-be looking for technical correctness of the documentation, adherence to the
-[style guide](https://developers.google.com/style/), good spelling and grammar
-(writing quality), intuitive organization, and strong documentation usability.
-Members should be proficient in at least one of these review areas.
+# Workflow
 
-### Requirements
+There are roughly two workflows: writing / contributing new content, and
+refining and reviewing existing content.
 
-- Has made multiple contributions to the project or community. Contributions
-  might include, but are not limited to:
+## New Content
 
-  - Authoring and reviewing PRs on GitHub in the Docs or Website repos.
+We expect most new content to be written by the subject matter expert (SME)
+which would be the contributor who actually worked on the feature or example.
 
-  - Filing and commenting on issues on GitHub.
+When writing new content, keep the following in mind:
 
-  - Contributing to working group or community discussions.
+- Write one [type of documentation](#type of documentation) per page.
 
-- Subscribed to
-  [knative-dev@googlegroups.com](https://groups.google.com/forum/#!forum/knative-dev).
+- If possible, follow the style/template of other documents of the same type.
 
-- Actively contributing to 1 or more areas.
+- Focus mostly on technical correctness; structure and language should be
+  roughly correct, but don't need heavy review in this phase.
 
-- Sponsored by 1 approver.
+The goal of adding new content is to get technically correct documentation into
+the repo before it is lost. Tech Writers may provide some quick guidance on
+getting documentation into the correct location, but won't be providing a
+detailed list of items to change.
 
-  - Done by adding GitHub user to Knative organization.
+## Refining and Reviewing Content
 
-### Responsibilities and privileges
+Once the raw documentation has made it into the repo, tech writers and other
+communications experts will review and revise the documentation to make it
+easier to consume. This will be done as a second PR; it's often easier for the
+tech writers to clean up or rewrite a section of prose than to teach an engineer
+what to do, and most engineers trust the wordsmithing the tech writers produce
+more than their own.
 
-- Responsive to issues and PRs assigned to them.
+When revising the content, the tech writer will create a new PR and send it to
+the original author to ensure that the content is technically correct; they may
+also ask a few clarifying questions, or add details such as diagrams or notes if
+needed. It's not necessarily expected that tech writers will actually execute
+the steps of a tutorial — it's expected that the SME is responsible for a
+working tutorial or how-to.
 
-- Active owner of documents they have contributed (unless ownership is
-  explicitly transferred).
+# Tech Writer Guide
 
-  - Addresses bugs or issues in their documentation in a timely manner.
+## Type of documentation
```

Comment:
This section and the next are too difficult of an ask for non-writers. With the right questions something like the "PDRD" (sorry just making these up as i go), then we should be able to use that info to identify the most appropriate place (vs asking a SME to identify it for a writer). 

Its not an easy task to target an audience, have a complete understanding of the current organization of our content (trying to get a handle today is very challenging), and then identify where your new content fits in.

I think the UX WG might be able to help here and possibly define our future/ideal content structure (and it would be easier and better to incrementally move toward that goal), rather than continue to simply add to mounds of content as they are today.

Im okay leaving this here to say we have instructions but i can almost guarantee the results wont meet expectations.